clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -337,167 +346,159 @@ private void handleSchemaCommand() {
    */
   private void handleTableCommand() {
     String url = getUrl();
+    String auth = getAuth();
+    String userName = line.getOptionValue(GravitinoOptions.LOGIN);
     FullName name = new FullName(line);
     String metalake = name.getMetalakeName();
     String catalog = name.getCatalogName();
     String schema = name.getSchemaName();
 
     if (CommandActions.LIST.equals(command)) {
-      new ListTables(url, ignore, metalake, catalog, schema).handle();
+      new ListTables(url, ignore, auth, userName, metalake, catalog, schema).handle();
       return;
     }
 
     String table = name.getTableName();
 
     if (CommandActions.DETAILS.equals(command)) {
       if (line.hasOption(GravitinoOptions.AUDIT)) {
-        new TableAudit(url, ignore, metalake, catalog, schema, table).handle();
+        new TableAudit(url, ignore, auth, userName, metalake, catalog, schema, table).handle();
       } else if (line.hasOption(GravitinoOptions.INDEX)) {
-        new ListIndexes(url, ignore, metalake, catalog, schema, table).handle();
+        new ListIndexes(url, ignore, auth, userName, metalake, catalog, schema, table).handle();
       } else if (line.hasOption(GravitinoOptions.DISTRIBUTION)) {
-        new TableDistribution(url, ignore, metalake, catalog, schema, table).handle();
-      } else if (line.hasOption(GravitinoOptions.Partition)) {
-        new TablePartition(url, ignore, metalake, catalog, schema, table).handle();
+        new TableDistribution(url, ignore, auth, userName, metalake, catalog, schema, table)
+            .handle();
+      } else if (line.hasOption(GravitinoOptions.PARTITION)) {
+        new TablePartition(url, ignore, auth, userName, metalake, catalog, schema, table).handle();
       } else {
-        new TableDetails(url, ignore, metalake, catalog, schema, table).handle();
+        new TableDetails(url, ignore, auth, userName, metalake, catalog, schema, table).handle();
```

Comment:
A little bit concerned that the list of params is getting longer and longer.

Reply:
This is in another PR that has not been merged and is unrelated to this PR.